Encounter1/Resources/values/styles.xml

```diff
@@ -1,11 +1,18 @@
 <resources>
 
-    <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
-    </style>
+	<style name="EncounterMe.LightMode" parent="@style/Theme.AppCompat.NoActionBar">
```

Comment:
For me location list became too light (white letters on grey screen)
https://pasteboard.co/6RzIUrTgvuMU.png